### PR TITLE
Cash cassettes edit wizard

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Add the edit wizard to the cashbox screen PT1](https://trello.com/c/rdsAiK20/692-add-the-edit-wizard-to-the-cashbox-screen-pt1)

**Review by hellypi — APPROVED** [review]
Good stuff, that wizard 👌🏻👌🏻👌🏻

File: new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+                      className={classnames(
+                        classes.horizontalAlign,
+                        classes.lineAlignment
+                      )}>
+                      <Info1 noMargin className={classes.cashboxBills}>
+                        {machine?.cashbox}
+                      </Info1>
+                      <P noMargin>accepted bills</P>
+                    </div>
+                    {steps[step - 1].fiatAmount && (
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js]: When will the step get fiatAmounts? I don't see this being used anywhere else.

File: new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+  const getPercentage = values =>
+    R.clamp(
+      0,
+      100,
+      (100 *
+        (step === 2
+          ? values.cassette1Count ?? cassetteInfo.amount
+          : values.cassette2Count ?? cassetteInfo.amount)) /
+        cassetteCapacity
+    )
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js]:
```suggestion
    const cassetteCount = step === 2 ? values.cassette1Count : values.cassette2Count
    const value = cassetteCount ?? cassetteInfo.amount
    R.clamp(0, 100, (100 * (value / cassetteCapacity))
```

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js (hunk 2 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/pages/Maintenance/Wizard/WizardStep.js]: Good idea!

File: new-lamassu-admin/src/pages/Maintenance/CashCassettes.js [hunk 3]
```diff
@@ -103,7 +108,9 @@ const CashCassettes = () => {
   const locale = data?.config && fromNamespace('locale')(data.config)
   const fiatCurrency = locale?.fiatCurrency
 
-  const onSave = (...[, { id, cashbox, cassette1, cassette2 }]) => {
+  console.log('data', data)
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Maintenance/CashCassettes.js]: Remove the console.log

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**